connectors/sources/generic_database.py

```diff
@@ -77,6 +96,7 @@ def get_default_configuration(cls):
                 "label": "Database",
                 "type": "str",
             },
+            "tables": {"value": ALL_TABLES, "label": "Tables", "type": "list"},
```

Comment:
```suggestion
            "tables": {"value": ALL_TABLES, "label": "Comma-separated list of tables", "type": "list"},
```

Should we mention this for now since we support only strings?

Reply:
Will the label be rendered as the text directly above the input field or will the key be capitalized and rendered above the input field?